### ISSUE TITLE
Problem installing UpgradeMODX

### ISSUE THREAD

**Issue by hgw7m** [BobRay/UpgradeMODX#46]
I'm trying to install UpgradeMODX 1.5.5 via PackageManagement into an older MODX 2.2.8. There is an error in the console log, the installer is unable to create chunk and snippet. I attach the log. The installer is not proceeding from the extra info page.

I cleared the cache and wiped out core/cache directory, tried again, same result. Also after deleting the package and new installation.

I've tried with PHP 5.6, PHP 5.5 and PHP 5.4.

Thanks for any hints.

I already upgraded an even older version (2.2.3) with UpgradeMODX and was very happy as it had been working great.

[output-2018-07-30 1108.txt](https://github.com/BobRay/UpgradeMODX/files/2240480/output-2018-07-30.1108.txt)

**Comment by BobRay** [comment]
Does 2.2.8 have widgets?  Is there an UpgradeMODX resource? If so, try viewing it.

**Comment by hgw7m** [comment]
Yes it has, and the UpgradeMODX Widget is om the Dashboard. But the snippet and chunks are not installed, the console gives error message in red lines, but after them says installation was succesful:

`chunk_err_createArray
Error saving vehicle object of class modChunk; criteria: Array ( [name] => UpgradeMODXSnippetScriptSource )
chunk_err_createArray
Error saving vehicle object of class modChunk; criteria: Array ( [name] => UpgradeMODXTpl )
snippet_err_createArray
Error saving vehicle object of class modSnippet; criteria: Array ( [name] => UpgradeMODXWidget )
Could not install related objects with locally owned keys for vehicle object of class modCategory; criteria: Array ( [category] => UpgradeMODX )`

I'm installing as Super User.

Thanks again, Bob!
Best regards
Guenter

**Comment by BobRay** [comment]
I wonder if mod_security is preventing those from saving. That's the only thing I can think of that would cause your symptoms.

Can you edit and save existing snippets and chunks?

**Comment by hgw7m** [comment]
Yes, I can change snippets. But when creating a chunk or a snippet, it's not saved and MODX is saying: Snippet Name is invalid ... / chunk name is invalid. Strange...

**Comment by BobRay** [comment]
Try repairing all the tables in the DB and manually deleting all files in the core/cache directory.

**Comment by BobRay** [comment]
Is this still an issue?
